runtime/main.h

```diff
@@ -0,0 +1,18 @@
+//===-- runtime/main.cc -----------------------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//----------------------------------------------------------------------------//
+
+#ifndef FORTRAN_RUNTIME_MAIN_H_
+#define FORTRAN_RUNTIME_MAIN_H_
+
+namespace Fortran::runtime {
+extern int argc;
+extern const char **argv;
+extern const char **envp;
```

Comment:
Since these global are `extern` here, think you need to declare them in `main.cc` so that they live somewhere. Also, maybe this file could be called "globals.h" instead of "main.h", though maybe that would be an incentive to create more globals...